daemon/daemon_unix.go

```diff
@@ -472,28 +472,28 @@ func verifyContainerResources(resources *containertypes.Resources, sysInfo *sysi
 func (daemon *Daemon) getCgroupDriver() string {
 	cgroupDriver := cgroupFsDriver
 
-	// No other cgroup drivers are supported at the moment. Warn the
-	// user if they tried to set one other than cgroupfs
-	for _, option := range daemon.configStore.ExecOptions {
+	if daemon.usingSystemd() {
+		cgroupDriver = cgroupSystemdDriver
```

Comment:
runc checks and errors out with "oci runtime error: systemd cgroup flag passed, but systemd support for managing cgroups is not available."
Checking for driver availability in docker means an expensive DBus call. As is seems okay, since user will know that the daemon is started with incorrect options on the first container run. 
Thoughts?